Makefile

```diff
@@ -202,6 +202,22 @@ THIRD_PARTY_NOTICE_LICENSE_TYPE = microsoft-oss
 $(eval $(call CREATE_THIRD_PARTY_NOTICES_RULE,ThirdPartyNotices.txt,foundation,False,False))
 $(eval $(call CREATE_THIRD_PARTY_NOTICES_RULE,bin/$(CONFIGURATION)/lib/xamarin.android/ThirdPartyNotices.txt,$(THIRD_PARTY_NOTICE_LICENSE_TYPE),True,False))
 
+EXTERNAL_GIT_PATH=$(topdir)/external
+XA_PATH=$(topdir)
+prepare-external-git-dependencies:
+	$(call MSBUILD_BINLOG,prep-tasks) build-tools/xa-prep-tasks/xa-prep-tasks.csproj
+	$(call MSBUILD_BINLOG,external-checkout) build-tools/xa-prep-tasks/xa-prep-tasks.csproj \
```

Comment:
Current "convention", for lack of a better phrase, is that the `MSBUILD_BINLOG` macro should contain the current target, or some variation thereof.

As such, could these use `prep-external-tasks` and `prep-external-checkout`, or *something* to more closely associate the resulting `.binlog` files with this target?

Reply:
I've updated the file names for these binlog calls.